src/Serialization/GAP.jl

```diff
@@ -282,18 +276,17 @@ install_GAP_deserialization(:IsPcGroup,
             length(relord))
       fam = GAPWrap.ElementsFamily(GAPWrap.FamilyObj(F))
       rws = GAP.Globals.SingleCollector(F, GapObj(relord))::GapObj
-      reli = load_object(s, Vector, d[:power_reli], Int)
-      rels = load_object(s, Vector, d[:power_rels], (Vector{Int}, Int))
-      for k in 1:length(reli)
-        GAP.Globals.SetPower(rws, reli[k],
-          GapObj(GAPWrap.ObjByExtRep(fam, GapObj(rels[k]))))
+      for data_rel in d[:power_rels]
+        (i, elm) = load_object(s, Tuple{Int, Vector{Int}}, data_rel,
+                     [Int, [Vector{Int}, Int]])
+        GAP.Globals.SetPower(rws, i,
+          GapObj(GAPWrap.ObjByExtRep(fam, GapObj(elm))))
       end
-      reli = load_object(s, Vector, d[:comm_reli], (Vector{Int}, Int))
-      rels = load_object(s, Vector, d[:comm_rels], (Vector{Int}, Int))
-      for k in 1:length(reli)
-        (j, i) = reli[k]
+      for data_rel in d[:comm_rels]
+        (j, i, elm) = load_object(s, Tuple{Int, Int, Vector{Int}}, data_rel,
+                        [Int, Int, [Vector{Int}, Int]])
```

Comment:
Maybe? this needs to be checked
```suggestion
                        [Int, Int, Vector{Int}])
```

Reply:
or it might be this, not sure?
```suggestion
                        [Int, Int, [Vector, Int]])
```

Reply:
or?
```suggestion
                        [Int, Int, [Vector, Int]])
```

Reply:
Yes, the latter works. Thanks.

Concerning the proposal to specify just `Tuple` as the second argument of `load_object`: Yes, this also works.
On the other hand, the second argument is logically the right place for specifying the desired type of the result (if it would be possible to omit the fourth argument).

I am going to update the pull request.

Reply:
yes I also agree, I think the issue is there isn't currently a function that can break down the second argument into all its params in the right way so that I can load an arbitrary tuple. I will make an attempt at dealing with this in a later pull request.